cmd/vicadmin/vicadm.go

```diff
@@ -96,8 +90,13 @@ var (
 )
 
 func init() {
-	trace.Logger.Level = log.DebugLevel
 	defer trace.End(trace.Begin(""))
+	trace.Logger.Level = log.DebugLevel
+	// We don't want to run this as root.
+	// Setuid to "vicadmin"
+	ud := syscall.Getuid()
+	gd := syscall.Getgid()
+	log.Info(fmt.Sprintf("Current UID/GID = %d/%d", ud, gd))
```

Comment:
comment says we don't want to run this as root but we do nothing about it, how about checking them to see whether they are 0 and then panic?